lib/memstore.ts

```diff
@@ -48,27 +17,66 @@ export type MemoryCookieStoreIndex = {
   }
 }
 
+/**
+ * An in-memory {@link Store} implementation for {@link CookieJar}. This is the default implementation used by
+ * {@link CookieJar} and supports both async and sync operations. Also supports serialization, getAllCookies, and removeAllCookies.
+ * @public
+ */
 export class MemoryCookieStore extends Store {
+  /**
+   * This value is `true` since {@link MemoryCookieStore} implements synchronous functionality.
+   */
   override synchronous: boolean
```

Comment:
```suggestion
  override synchronous: true
```
Not super important, but does this work? 🤷

Reply:
No, it breaks the tests.